### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by andreasknoepfle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v0.7.0
+
+- Added for `link_fn` option in `button_ui` to support LiveView's `live_path`/`live_redirect`
```
Comment (andreasknoepfle) [inline CHANGELOG.md:7]:
```suggestion
- Added for `link_fn` option in `button_ui` to support LiveView's `live_patch`/`live_redirect`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v0.7.0
+
+- Added for `link_fn` option in `button_ui` to support LiveView's `live_patch`/`live_redirect`
```
Comment (andreasknoepfle) [inline CHANGELOG.md:7]:
```suggestion
- Added`link_fn` option in `button_ui` to support LiveView's `live_patch`/`live_redirect`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v0.7.0
+
+- Added `link_fn` option in `button_ui` to support LiveView's `live_patch`/`live_redirect`
```
Comment (agatheblues) [inline CHANGELOG.md:7]: Isn't it `ui_button`?

File: config/config.exs [hunk 4]
```diff
@@ -4,4 +4,4 @@ config :bitstyles_phoenix, :trim_e2e_classes, true
 
 config :bitstyles_phoenix,
        :bitstyles_dist,
-       "https://cdn.jsdelivr.net/npm/bitstyles@1.1.0"
+       "https://cdn.jsdelivr.net/npm/bitstyles@1.5.0"
```
Comment (andreasknoepfle) [inline config/config.exs:7]: This is a flyby, so that we pick the proper version for the doc examples.

**Review by agatheblues — APPROVED** [review]
🪐

File: CHANGELOG.md (hunk 3 above)
Comment (andreasknoepfle) [reply CHANGELOG.md:7]: 🤦  how hard can it be to write a good changelog :D

Comment (andreasknoepfle) [inline CHANGELOG.md:7]:
```suggestion
- Added `link_fn` option in `ui_button` to support LiveView's `live_patch`/`live_redirect`
```

**Pull request merged**